action.yml

```
@@ -32,8 +32,10 @@ runs:
-              --message "${{ inputs.message }}" \
-              --access-token "${{ inputs.access-token }}" \
-              --pr-title "${{ inputs.pr-title }}" \
-              ${{ inputs.base-url && format('--base-url {0}', inputs.base-url) || '' }}
\ No newline at end of file
+          python3 ${{ github.action_path }}/post_to_mastodon.py \
+          --message "${{ inputs.message }}" \
+          --access-token "${{ inputs.access-token }}" \
+          --pr-title "${{ inputs.pr-title }}" \
+          ${{ if inputs.base-url }}
+            --base-url "${{ inputs.base-url }}"
+          ${{ endif }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)